data/json/npcs/BGSS_talk_common.json

```diff
@@ -0,0 +1,704 @@
+[{
+    "id": "TALK_LEADER",
+    "type": "talk_topic",
+    "responses": [
+	{ "text": "<BGSS_intro_question>", 
```

Comment:
I can't find where you actually define this snippet.

Reply:
I haven't yet. Snippets upcoming shortly.